dmoj/graders/bridged.py

```diff
@@ -0,0 +1,72 @@
+import os
+import subprocess
+
+from dmoj.contrib import contrib_modules
+from dmoj.error import InternalError
+from dmoj.graders.standard import StandardGrader
+from dmoj.judgeenv import env, get_problem_root
+from dmoj.utils.aux_files import compile_with_auxiliary_files, mktemp
+from dmoj.utils.unicode import utf8text
+
+
+class BridgedInteractiveGrader(StandardGrader):
+    def __init__(self, judge, problem, language, source, meta):
+        super().__init__(judge, problem, language, source, meta)
+        self.handler_data = self.problem.config.interactive
+        self.interactor_binary = self._generate_interactor_binary()
+
+    def check_result(self, case, result):
+        if result.result_flag:
+            # This is usually because of a TLE verdict raised after the interactor
+            # has issued the AC verdict
+            # This results in a TLE verdict getting full points, which should not be the case
+            return False
+
+        stderr = self._interactor.stderr.read()
+        returncode = self.handler_data.get('returncode', 'default')
```

Comment:
`returncode` is used in some parts of this code and `return_code` in others. (`return_code` is probably better everywhere.)